.gitignore

```diff
@@ -32,3 +32,4 @@ configs/example/memcheck.cfg
 configs/dram/lowp_sweep.cfg
 .pyenv
 .vscode
+gem5_build
```

Comment:
No, the users can still use the `build` directory to build gem5. The `gem5_build` directory is used for kconfig to avoid conflict with old scheme in `build` directory.

Reply:
Removed